repository/src/main/java/tech/ydb/yoj/repository/db/AbstractDelegatingTable.java

```diff
@@ -117,11 +117,6 @@ public Class<T> getType() {
         return target.getType();
     }
 
```

Comment:
Missing `find(Set<ID>)` implementation in `AbstractDelegatingTable`:
```java
    @Override
    public <ID extends Entity.Id<T>> List<T> find(Set<ID> ids) {
        return target.find(ids);
    }
```

Reply:
it's already implemented